docs/source/en/model_doc/seggpt.md

```diff
@@ -26,7 +26,8 @@ The abstract from the paper is the following:
 
 Tips:
 - One can use [`SegGptImageProcessor`] to prepare image input, prompt and mask to the model.
-- It's highly advisable to pass `num_labels` (not considering background) during preprocessing and postprocessing with [`SegGptImageProcessor`] for your use case.
+- When using binary masks or segmentation maps as prompts pass it to [`SegGptImageProcessor`] `segmentation_maps` argument, if you already converted your prompt mask to RGB then pass it to `prompt_masks` argument.
```

Comment:
To be updated

Reply:
https://github.com/huggingface/transformers/pull/29550/commits/017bd48b4e9c51ea8161d757d47a89ed7c7e8ca2